src/itemdb.d

```diff
@@ -29,7 +31,7 @@ struct Item {
 final class ItemDatabase
 {
 	// increment this for every change in the db schema
-	immutable int itemDatabaseVersion = 6;
+	immutable int itemDatabaseVersion = 7;
```

Comment:
I was just thinking about this - what's the reason for incrementing the database version at this point?  There's no structural changes that I can see?

Reply:
The reason is a combination of factors actually:
* When folks are migrating from the 'skilion' code to this source tree, due to the number of issues that the 'skilion' codebase has, the code comes back and complains that the parent item is not in the database, when it actually is, then the code does not update the items correctly.
* By regenerating the database, it also enables WAL for database performance enhancements

The best way previously to fix this was to issue a `--resync` command which would remove the local database file and re-create it. Additionally, when suggesting to the Redhat / CentOS / Fedora folk to point to my repo, they have issues in 'deleting' a user file in their home directory (RHBZ #1598934) so I saw this as a way to resolve this issue and the prior 2 in one change.

Whilst technically no schema change - ensuring that all the data is correct in the DB is what the increment will facilitate.

Reply:
excellent - looks good to me 👍